install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
+    curl -s -A "$CURL_USER_AGENT" -f "$URL.sig" > /var/tmp/sandstorm-$BUILD.tar.xz.sig
```

Comment:
Same suggestion re: don't use predictable filename. Partly because using a predictable filename means that if two local users on the same machine try to install Sandstorm (e.g. planning to play with it in their $HOME), the second one will get "Permission denied" and I have run into that installing `pip` and it makes me so rageful to think that the `pip` developers back then didn't use multi-user UNIX systems and I hope we can be better than that.